<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackWhiteAdjustForm</class>
 <widget class="QWidget" name="BlackWhiteAdjustForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>254</width>
     <height>362</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>红色：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:red</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="redEdit">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="redSlider">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
border: 1px solid grey;
height: 4px;
margin:0 6 0 6;
border-radius:3px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 black, stop:0.6 red, stop:1 white);

}
QSlider::handle:horizontal   
{  
    border-image:url(:/images/handle.png);
    width: 12px;
    margin:1 -6 -7 -6;
}  </string>
        </property>
        <property name="minimum">
         <number>-200</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>黄色：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:yellow</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="yellowEdit">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="yellowSlider">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
border: 1px solid grey;
height: 4px;
margin:0 6 0 6;
border-radius:3px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 black, stop:0.6 yellow, stop:1 white);

}
QSlider::handle:horizontal   
{  
    border-image:url(:/images/handle.png);
    width: 12px;
    margin:1 -6 -7 -6;
}  </string>
        </property>
        <property name="minimum">
         <number>-200</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>绿色：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:green</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="greenEdit">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="greenSlider">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
border: 1px solid grey;
height: 4px;
margin:0 6 0 6;
border-radius:3px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 black, stop:0.6 green, stop:1 white);

}
QSlider::handle:horizontal   
{  
    border-image:url(:/images/handle.png);
    width: 12px;
    margin:1 -6 -7 -6;
}  </string>
        </property>
        <property name="minimum">
         <number>-200</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>青色：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:cyan</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cyanEdit">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="cyanSlider">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
border: 1px solid grey;
height: 4px;
margin:0 6 0 6;
border-radius:3px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 black, stop:0.6 cyan, stop:1 white);

}
QSlider::handle:horizontal   
{  
    border-image:url(:/images/handle.png);
    width: 12px;
    margin:1 -6 -7 -6;
}  </string>
        </property>
        <property name="minimum">
         <number>-200</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>蓝色：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:blue</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="blueEdit">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="blueSlider">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
border: 1px solid grey;
height: 4px;
margin:0 6 0 6;
border-radius:3px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 black, stop:0.6 blue, stop:1 white);

}
QSlider::handle:horizontal   
{  
    border-image:url(:/images/handle.png);
    width: 12px;
    margin:1 -6 -7 -6;
}  </string>
        </property>
        <property name="minimum">
         <number>-200</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>洋红：</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:magenta</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="magentaEdit">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="magentaSlider">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
border: 1px solid grey;
height: 4px;
margin:0 6 0 6;
border-radius:3px;
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 black, stop:0.6 magenta, stop:1 white);

}
QSlider::handle:horizontal   
{  
    border-image:url(:/images/handle.png);
    width: 12px;
    margin:1 -6 -7 -6;
}  </string>
        </property>
        <property name="minimum">
         <number>-200</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
